contents/docs/integrate/_snippets/install-flutter.mdx

```diff
@@ -6,63 +6,116 @@ Set your PostHog API key and change the automatic event tracking on if you wish
 
 Remember that the application lifecycle events won't have any special context set for you by the time it is initialized. If you are using a self-hosted instance of PostHog you will need to have the public hostname or IP for your instance as well.
 
-#### Android
+To start, add `posthog_flutter` to your `pubspec.yaml`:
 
-```xml file=AndroidManifest.xml
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.posthog.posthog_flutter_example">
+```yaml file=pubspec.yaml
+# rest of your code
+
+dependencies:
+  flutter:
+    sdk: flutter
+  posthog_flutter: ^4.0.1
+
+# rest of your code
+```
+
+Then complete the set up for each platform:
+
+#### Android setup
+
+For Android, add your PostHog configuration to your `AndroidManifest.xml` file located in the `android/app/src/main`:
+
+```xml file=android/app/src/main/AndroidManifest.xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="your.package.name">
     <application>
-        <activity>
-            [...]
-        </activity>
+        <!-- ... other configuration ... -->
         <meta-data android:name="com.posthog.posthog.API_KEY" android:value="<ph_project_api_key>" />
-        <!-- usually 'https://app.posthog.com' or 'https://eu.posthog.com' -->
         <meta-data android:name="com.posthog.posthog.POSTHOG_HOST" android:value="<ph_instance_address>" />
         <meta-data android:name="com.posthog.posthog.TRACK_APPLICATION_LIFECYCLE_EVENTS" android:value="true" />
         <meta-data android:name="com.posthog.posthog.DEBUG" android:value="true" />
     </application>
 </manifest>
 ```
 
-#### iOS/macOS
+/Users/lior/Desktop/flutter_ab_tests_2/flutter_ab_tests_2/
 
-```xml file=Info.plist
+You'll also need to update the minimum Android SDK version to `21` in `android/app/build.gradle`:
```

Comment:
the latest flutter version is already 21 https://github.com/flutter/flutter/blob/d8503febb10f42240aec922d2707b2c479a75310/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy#L47-L48
did you need to set it up anyway?